pkg/apis/cnsoperator/cnsvolumemetadata/v1alpha1/cnsvolumemetadata_types.go

```diff
@@ -71,6 +71,11 @@ type CnsVolumeMetadataSpec struct {
 	// namespace is set for objects whose EntityType is PERSISTENT_VOLUME.
 	//+optional
 	Namespace string `json:"namespace,omitempty"`
+
+	// K8sDistribution indicates the kubernetes distribution where the
```

Comment:
Can you update this comment to match the variable name? Also use "TKG" instead of "guest"

Reply:
Done.